### PR TITLE
Fix some bugs and implement new primitive in StartReplay struct

### PR DESCRIPTION
**Pull request by WhiteTPoison**

### CODE REVIEW

File: Codes/SlippiReplays/Record.h [hunk 1]
```diff
@@ -13,5 +13,9 @@
 #include <Brawl/IT/itManager.h>
 #include "EXIPacket.h"
 #include "brawlback-exi-structures/ExiStructures.h"
+#include <etl/string.h>
```
Comment (eliasreid) [inline Codes/SlippiReplays/Record.h:16]: did you mean to add these includes to Record.cpp?
either way, chrono and etl/string should no longer be needed

File: Libraries/Brawl/IT/BaseItem.h [hunk 2]
```diff
@@ -2,6 +2,7 @@
 #ifndef PROJECTMCODES_BASEITEM_H
 #define PROJECTMCODES_BASEITEM_H
 
+#include "stdtypes.h"
```
Comment (eliasreid) [inline Libraries/Brawl/IT/BaseItem.h:5]: Was this needed to fix a compilation error - same question for itManager.h

File: Libraries/etl/memory.h [hunk 3]
```diff
@@ -45,6 +45,8 @@ SOFTWARE.
 
 #if defined(ETL_IN_UNIT_TEST) || ETL_USING_STL
   #include <memory>
+#include <cstring>
```
Comment (eliasreid) [inline Libraries/etl/memory.h:48]: this was needed to fix a compile error? seems weird we'd have to add includes to etl headers

File: Codes/SlippiReplays/EXIPacket.h [hunk 4]
```diff
@@ -9,7 +9,12 @@ enum EXICommand
 {
     START_REPLAYS_STRUCT = 19,
     REPLAYS_STRUCT = 20,
-    ENDGAME = 21
+    ENDGAME = 21,
```
Comment (eliasreid) [inline Codes/SlippiReplays/EXIPacket.h:12]: Maybe out of scope for this review, but these values are shared between dolphin / game, so belong in the exi-structures repo IMO

File: Libraries/Brawl/IT/BaseItem.h (hunk 2 above)
Comment (JaredWhiteOne) [reply Libraries/Brawl/IT/BaseItem.h:5]: Yes.

**Pull request merged**